lib/src/showcase/showcase_view.dart

```diff
@@ -424,7 +424,12 @@ class ShowcaseView {
 
     if (autoPlay) {
       _cancelTimer();
-      _timer = Timer(autoPlayDelay, () => next(force: true));
+      // Showcase is first.
+      final config = _getCurrentActiveControllers[_activeWidgetId!];
```

Comment:
The list returned by `_getCurrentActiveControllers` is different from the `_ids` list so `_activeWidgetId` can't be used to index it. 

Here, you can use the first controller from the list.
```suggestion
      final config = _getCurrentActiveControllers.firstOrNull;
```
PS: The list is to support multiple showcase at the same time and in such case, the package takes config of the first widget.